man/dockerd.8.md

```diff
@@ -441,7 +441,7 @@ device. And devices automatically go away when last user of the device
 exits.
 
 For example, when a container exits, its associated thin device is removed. If
-that device has leaked into some other mount namespace and can't be removed,
+that device has leaked into some other mount namespaces and can't be removed,
```

Comment:
shouldn't it be `some other mount's namespace` ?
could also be `leaked into some other mount-namespace`